_posts/en/newsletters/2019-01-08-newsletter.md

```diff
@@ -0,0 +1,168 @@
+    but the cost of owning the bitcoins Mallory would've traded.
+
+    There currently aren't any known cross-currency LN nodes, but the
+    availability of this trick means that future such nodes could be
+    abused for speculation rather than payment routing.  If this turns
+    out to be a real problem and if an acceptable solution isn't found,
+    it may be the case that payment channel networks for different
+    currencies will be isolated from each other.
+
+## Notable code changes
+
+*Notable code changes this week in [Bitcoin Core][core commits],
+[LND][lnd commits], [C-lightning][cl commits], and [libsecp256k1][secp
+commits].*
+
+- [Bitcoin Core #14565][] significantly improves the error handling for
+  the `importmulti` RPC and will return a `warnings` field for each
+  attempted import with an array of strings describing any problems with
+  the that import (but only if there were any problems).
+
+- [Bitcoin Core #14811][] updates the [getblocktemplate][rpc
+  getblocktemplate] RPC to require that the segwit flag be passed.  This
+  helps prevent miners from accidentally not using segwit, which reduces
+  their fee income.  See [Newsletter #20][] for a recent instance where
+  this may have happened to a large mining pool.
+
+- [Bitcoin Core #14336][] changes the system call (syscall) Bitcoin Core
+  uses to handle network connections and other resources on platforms
+  that support an alternative syscall (only Linux for now).  On a
+  Debian GNU/Linux x86_64 system tested by Optech, this increased the
+  maximum allowed number of connections from 865 to 9,999.  Please note,
+  developers strongly advise against increasing the maximum number of
+  connections unless you have a special need for more connections.  For
+  each peer you have after your first peer, you increase the amount of
+  bandwidth overhead used by your node and by all of your peers.  The
+  default minimum of 8 and maximum of 125 peers were chosen to provide
+  enough connections to ensure robust relaying but also to not create so
+  much bandwidth overhead that the network would be excessively
+  wasteful.  As relay improvements are implemented (see the News section
+  of [Newsletter #26][]), Bitcoin Core developers may adjust these
+  defaults.
+
+- [C-Lightning #2172][] allows `lightningd` to be shutdown normally even
+  if it's operating as the primary process (PID 1), which can be useful
+  in Docker containers.
+
+- [C-Lightning #2188][] adds notification subscription handlers that can
+  be used by plugins, with initial support for notifications that the
+  node has connected to a new peer or disconnected from an existing
+  peer.  The [plugin documentation][cl plugin event] and [sample
+  plugin][cl helloworld.py] have been updated for these handlers.
+
+- [LND #2374][] increases the maximum size of the gRPC messages the
+  `lncli` tool will accept from 4 MB to 50 MB.  This fixes a problem
```

Comment:
minor nit: I found the "will accept from..." tripped me up on my first parse - the *from* attached itself to *accept* instead of *increases*. You could consider parenthesizing *from 4 MB to 50 MB* (although I'll admit I'm often too liberal with parentheses)